projects/netgrif-components/src/lib/header/header.theme.scss

```diff
@@ -6,14 +6,14 @@
         background: rgba(mat.get-color-from-palette($primary), 0.3);
     }
 
-    .header-color {
-        background: mat.get-color-from-palette($primary, 500);
-        color: mat.get-contrast-color-from-palette($primary, 500);
-    }
-
-    .header-color-font {
-        color: mat.get-contrast-color-from-palette($primary, 500);
-    }
+    //.header-color {
+    //    background: mat.get-color-from-palette($primary, 500);
+    //    color: mat.get-contrast-color-from-palette($primary, 500);
+    //}
+    //
+    //.header-color-font {
+    //    color: mat.get-contrast-color-from-palette($primary, 500);
+    //}
```

Comment:
We should remove unused or commented code.

Reply:
done